packages/ui-react/src/components/Card/Card.module.scss

```diff
@@ -29,18 +29,17 @@
 
   &.featured {
     .title {
-      height: 1.1em;
+      height: 1.3em;
```

Comment:
Should this be `height: 1.5em` or even `height: variables.$base-line-height` to link it (the height depends on the line height here)